pandas/tests/indexes/test_multi.py

```diff
@@ -766,6 +766,14 @@ def test_from_arrays_empty(self):
                 ValueError, "Must pass non-zero number of levels/labels"):
             MultiIndex.from_arrays(arrays=[])
 
+        # 0 levels, names defined
+        result = MultiIndex.from_arrays(arrays=[],
```

Comment:
I think ``arrays=[]`` is reasonable here (can you add a test for both ``[[], []]`` and ``[]`` (for ``.from_arrays``)